Marlin/Marlin_main.cpp

```diff
@@ -868,7 +872,28 @@ void setup() {
  *  - Call LCD update
  */
 void loop() {
-  if (commands_in_queue < BUFSIZE) get_available_commands();
+
+
+if (filamentRemove) {
```

Comment:
@jbrazio aparently i'm no being able to reset the configuration_store.cpp file to the original one.
i'm tryng `git reset c86d092 Marlin/configuration_store.cpp` but i had no success, the files remains the one changed
 well is best i replace the whole file with the one used in the rcbugfix? It will appears like it has been change a lot but in fact it wont.